src/views/admin/PersonDetailsView.vue

```diff
@@ -1316,31 +1316,35 @@
       :padded="true"
       :showCloseText="true"
     >
-      <!-- Error Message Display if the personStore throws any kind of error (Not being able to load the person) -->
-      <SpshAlert
-        :model-value="!!personStore.errorCode"
-        :type="'error'"
-        :closable="false"
-        :text="$t(`admin.person.errors.${personStore.errorCode}`)"
-        :showButton="true"
-        :buttonText="alertButtonTextKopers"
-        :buttonAction="alertButtonActionKopers"
-        :title="$t(`admin.person.title.${personStore.errorCode}`)"
-        @update:modelValue="handleAlertClose"
-      />
-
-      <!-- Error Message Display if the personenkontextStore throws any kind of error (Not being able to load the kontext) -->
-      <SpshAlert
-        :model-value="!!personenkontextStore.errorCode"
-        :type="'error'"
-        :closable="false"
-        :text="creationErrorText"
-        :showButton="true"
-        :buttonText="alertButtonText"
-        :buttonAction="alertButtonAction"
-        :title="creationErrorTitle"
-        @update:modelValue="handleAlertClose"
-      />
+      <v-container class="px-3 px-sm-16">
```

Comment:
Dieses Wrapping resultiert in sowas:

![image](https://github.com/user-attachments/assets/70a343aa-ccf4-4b0a-9a73-f5c6567eaeea)

Reply:
done